content/mooreslaw-tutorial.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<p>In <code>np.exp</code>  I think you copied the explanation for the <code>log</code>  function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I wouldn't put it past myself, but I think the latest commit has the correct links. _I'm copy-pasting the markdown to double-check_:
* [`np.log`](https://numpy.org/doc/stable/reference/generated/numpy.log.html): this function takes the natural log of a NumPy array
* [`np.exp`](https://numpy.org/doc/stable/reference/generated/numpy.exp.html): this function takes the natural log of a NumPy array

Reply:
<p>The links are right, but np.exp takes the exponential of a NumPy array, right? :) </p>

Reply:
haha, yes `exp` does not take the "natural log of a NumPy array". Your comment said exactly that, whoops.